frameworks/elastic/tests/test_sanity.py

```diff
@@ -1,5 +1,3 @@
-import logging
```

Comment:
Oh importing everything from `tests.config` includes an import of `logging`

That's not immediately clear, hence we should do a cleanup in a separate PR.